src/App.svelte

```diff
@@ -69,11 +69,12 @@
   // Detect changes in hash and update settings appropriately
   window.addEventListener('hashchange', () => {
     if (!writingHash && hashShouldUpdate()) {
-      settings = getSettings(config);
+      const nextSettings = getSettings(config);
+      settings = nextSettings;
 
       // Update mapsStore if necessary
       if (settings.maps.length) {
-        const newMaps = maps.map((map, index) => ({
+        const newMaps = settings.maps.map((map, index) => ({
```

Comment:
Same as https://github.com/stamen/maperture/pull/142/files#r1063763146, want to make sure I'm not misunderstanding the existing logic here!

Reply:
I think this was just outdated logic.